Tests/SonarScanner.MSBuild.PreProcessor.Test/SonarQubeWebServerTest.cs

```diff
@@ -28,532 +28,550 @@
 using TestUtilities;
 
-namespace SonarScanner.MSBuild.PreProcessor.Test
+namespace SonarScanner.MSBuild.PreProcessor.Test;
+
+[TestClass]
+public class SonarQubeWebServerTest
 {
-    [TestClass]
-    public class SonarQubeWebServerTest
+    private const string ProjectKey = "project-key";
+    private const string ProjectBranch = "project-branch";
+
+    private readonly TestDownloader downloader;
+    private readonly Version version;
+    private readonly TestLogger logger;
+
+    private SonarQubeWebServer sut;
+
+    public SonarQubeWebServerTest()
     {
-        private const string ProjectKey = "project-key";
-        private const string ProjectBranch = "project-branch";
+        downloader = new TestDownloader();
+        version = new Version("9.9");
+        logger = new TestLogger();
+    }
 
-        private readonly TestDownloader downloader;
-        private readonly Version version;
-        private readonly TestLogger logger;
+    [TestInitialize]
+    public void Init() =>
+        sut = new SonarQubeWebServer(downloader, version, logger, null);
+
+    [TestCleanup]
+    public void Cleanup() =>
+        sut?.Dispose();
+
+    [DataTestMethod]
+    [DataRow("7.9.0.5545", false)]
+    [DataRow("8.0.0.18670", false)]
+    [DataRow("8.8.0.1121", false)]
+    [DataRow("8.9.0.0", true)]
+    [DataRow("9.0.0.1121", true)]
+    [DataRow("10.15.0.1121", true)]
+    public void IsServerVersionSupported(string sqVersion, bool expected)
+    {
+        sut = new SonarQubeWebServer(Substitute.For<IDownloader>(), new Version(sqVersion), logger, null);
 
-        private SonarQubeWebServer sut;
+        sut.IsServerVersionSupported().Should().Be(expected);
+    }
 
-        public SonarQubeWebServerTest()
-        {
-            downloader = new TestDownloader();
-            version = new Version("9.9");
-            logger = new TestLogger();
-        }
-
-        [TestInitialize]
-        public void Init() =>
-            sut = new SonarQubeWebServer(downloader, version, logger, null);
-
-        [TestCleanup]
-        public void Cleanup() =>
-            sut?.Dispose();
-
-        [DataTestMethod]
-        [DataRow("7.9.0.5545", false)]
-        [DataRow("8.0.0.18670", false)]
-        [DataRow("8.8.0.1121", false)]
-        [DataRow("8.9.0.0", true)]
-        [DataRow("9.0.0.1121", true)]
-        [DataRow("10.15.0.1121", true)]
-        public void IsServerVersionSupported(string sqVersion, bool expected)
-        {
-            sut = new SonarQubeWebServer(Mock.Of<IDownloader>(), new Version(sqVersion), logger, null);
+    [DataTestMethod]
+    [DataRow("{ }")]
+    [DataRow(@"{ ""isValidLicense"": false }")]
+    public async Task IsServerLicenseValid_Commercial_AuthNotForced_LicenseIsInvalid(string responseContent)
+    {
+        var response = new HttpResponseMessage { StatusCode = HttpStatusCode.OK, Content = new StringContent(responseContent) };
+        var downloaderMock = Substitute.For<IDownloader>();
```

Comment:
This doesn't need to be called `Mock` anymore with the new lib
```suggestion
        var downloader = Substitute.For<IDownloader>();
```